### PR TITLE
Add version: VPNetwork.TorGuard version 4.8.9

### PR DESCRIPTION
**Pull request by russellbanks**
- [X] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [X] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [X] Have you tested your manifest locally with `winget install --manifest <path>`?
- [X] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/80211)

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]

**Comment by russellbanks** [comment]
.

File: manifests/v/VPNetwork/TorGuard/4.8.9/VPNetwork.TorGuard.installer.yaml [hunk 1]
```diff
@@ -12,17 +12,15 @@ InstallModes:
 - interactive
 - silent
 UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: VPNetwork LLC - TorGuard - Online Privacy Protection Services
+  DisplayVersion: 4.8.9-build.144.1+g3174a6b
+  Publisher: VPNetwork LLC
```
Comment (ItzLevvie) [inline manifests/v/VPNetwork/TorGuard/4.8.9/VPNetwork.TorGuard.installer.yaml:18]:
```suggestion
  Publisher: '"VPNetwork LLC"'
```

File: manifests/v/VPNetwork/TorGuard/4.8.9/VPNetwork.TorGuard.installer.yaml [hunk 2]
```diff
@@ -12,17 +12,15 @@ InstallModes:
 - interactive
 - silent
 UpgradeBehavior: install
+AppsAndFeaturesEntries:
+- DisplayName: VPNetwork LLC - TorGuard - Online Privacy Protection Services
+  DisplayVersion: 4.8.9-build.144.1+g3174a6b
```
Comment (ItzLevvie) [inline manifests/v/VPNetwork/TorGuard/4.8.9/VPNetwork.TorGuard.installer.yaml:17]:
```suggestion
  DisplayVersion: '"4.8.9-build.144.1+g3174a6b"'
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
See https://github.com/microsoft/winget-pkgs/pull/80209#pullrequestreview-1127524702

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**